src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Extensions/ServiceClaimExtensionsTests.cs

```diff
@@ -0,0 +1,67 @@
+        [TestCase(ServiceClaim.DAV, new ServiceClaim[] { ServiceClaim.DAA, ServiceClaim.DAB, ServiceClaim.DAC, ServiceClaim.DAV }, true)]
+        [TestCase(ServiceClaim.DAA, new ServiceClaim[] { ServiceClaim.DAB, ServiceClaim.DAC, ServiceClaim.DAV }, false)]
+        [TestCase(ServiceClaim.DAB, new ServiceClaim[] { ServiceClaim.DAB, ServiceClaim.DAC, ServiceClaim.DAV }, true)]
+        [TestCase(ServiceClaim.DAC, new ServiceClaim[] { ServiceClaim.DAB, ServiceClaim.DAC, ServiceClaim.DAV }, true)]
+        [TestCase(ServiceClaim.DAV, new ServiceClaim[] { ServiceClaim.DAB, ServiceClaim.DAC, ServiceClaim.DAV }, true)]
+        [TestCase(ServiceClaim.DAA, new ServiceClaim[] { ServiceClaim.DAC, ServiceClaim.DAV }, false)]
+        [TestCase(ServiceClaim.DAB, new ServiceClaim[] { ServiceClaim.DAC, ServiceClaim.DAV }, false)]
+        [TestCase(ServiceClaim.DAC, new ServiceClaim[] { ServiceClaim.DAC, ServiceClaim.DAV }, true)]
+        [TestCase(ServiceClaim.DAV, new ServiceClaim[] { ServiceClaim.DAC, ServiceClaim.DAV }, true)]
+        [TestCase(ServiceClaim.DAA, new ServiceClaim[] { ServiceClaim.DAA }, true)]
+        [TestCase(ServiceClaim.DAB, new ServiceClaim[] { ServiceClaim.DAA }, true)]
+        [TestCase(ServiceClaim.DAC, new ServiceClaim[] { ServiceClaim.DAA }, true)]
+        [TestCase(ServiceClaim.DAV, new ServiceClaim[] { ServiceClaim.DAA }, true)]
+        [TestCase(ServiceClaim.DAA, new ServiceClaim[] { ServiceClaim.DAB }, false)]
+        [TestCase(ServiceClaim.DAB, new ServiceClaim[] { ServiceClaim.DAB }, true)]
+        [TestCase(ServiceClaim.DAC, new ServiceClaim[] { ServiceClaim.DAB }, true)]
+        [TestCase(ServiceClaim.DAV, new ServiceClaim[] { ServiceClaim.DAB }, true)]
+        [TestCase(ServiceClaim.DAA, new ServiceClaim[] { ServiceClaim.DAC }, false)]
+        [TestCase(ServiceClaim.DAB, new ServiceClaim[] { ServiceClaim.DAC }, false)]
+        [TestCase(ServiceClaim.DAC, new ServiceClaim[] { ServiceClaim.DAC }, true)]
+        [TestCase(ServiceClaim.DAV, new ServiceClaim[] { ServiceClaim.DAC }, true)]
+        [TestCase(ServiceClaim.DAA, new ServiceClaim[] { ServiceClaim.DAV }, false)]
+        [TestCase(ServiceClaim.DAB, new ServiceClaim[] { ServiceClaim.DAV }, false)]
+        [TestCase(ServiceClaim.DAC, new ServiceClaim[] { ServiceClaim.DAV }, false)]
+        [TestCase(ServiceClaim.DAV, new ServiceClaim[] { ServiceClaim.DAV }, true)]
+        [TestCase(ServiceClaim.DAA, new ServiceClaim[] { }, false)]
+        [TestCase(ServiceClaim.DAB, new ServiceClaim[] { }, false)]
+        [TestCase(ServiceClaim.DAC, new ServiceClaim[] { }, false)]
+        [TestCase(ServiceClaim.DAV, new ServiceClaim[] { }, false)]
+        public void ShouldReturnHasPermission(ServiceClaim minimumServiceClaim, ServiceClaim[] actualServiceClaims, bool expected)
+        {
+            var claims = actualServiceClaims.Select(a => new Claim(ProviderClaims.Service, a.ToString()));
```

Comment:
can we add some none das claims to this list such as DCFT to cover the user having Service claims but none being valid for our service?